src/rules/require-tothrow-message.js

```diff
@@ -1,41 +1,13 @@
-import { argument, expectCase, getDocsUrl, method } from './util';
+import requireToThrowMessage from './require-to-throw-message';
 
-export default {
-  meta: {
-    docs: {
-      url: getDocsUrl(__filename),
-    },
-    messages: {
-      requireRethrow: 'Add an error message to {{ propertyName }}()',
-    },
-    schema: [],
-  },
-  create(context) {
-    return {
-      CallExpression(node) {
-        if (!expectCase(node)) {
-          return;
-        }
+const replacedBy = ['require-to-throw-message'];
 
-        let targetNode = method(node);
-        if (targetNode.name === 'rejects') {
-          targetNode = method(node.parent);
-        }
+const requireTothrowMessage = Object.assign({}, requireToThrowMessage, {
```

Comment:
you can use object spread (we transpile with babel)

Reply:
I didn't see this repo was transpiled...I was waiting for the TS conversion to do this. I'll make that change.

Reply:
We use `import` & `export` 🙂 And yeah, I landed the Babel change to make the TS migration have a smaller diff

Reply:
Updated to object spread